weave/ops_domain/artifact_version_ops.py

```diff
@@ -544,21 +545,33 @@ def history_metrics(
 
 # TODO: Move all this to helper functions off the artifactVersion object
 def _artifact_version_to_wb_artifact(artifactVersion: wdt.ArtifactVersion):
-    entity_name = artifactVersion["artifactSequence"]["defaultArtifactType"]["project"][
-        "entity"
-    ]["name"]
-    project_name = artifactVersion["artifactSequence"]["defaultArtifactType"][
-        "project"
-    ]["name"]
+    print(f"\n\nlogging version query result: ===> \n{artifactVersion}\n\n", flush=True)
+    artifact_id = artifactVersion["id"]
     type_name = artifactVersion["artifactSequence"]["defaultArtifactType"]["name"]
     home_sequence_name = artifactVersion["artifactSequence"]["name"]
     commit_hash = artifactVersion["commitHash"]
+    uri = artifact_wandb.WeaveWBArtifactByIDURI(
```

Comment:
would be nice to have some commentary about why we sometimes use this class and other times use the other class

Reply:
Sure, will add!